recipes/nifty/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set version = "v1.0.8" %}
+{% set checksum = "4a70eb106a926271cf86cae3e568c71b26ca40ecdfbfc337110f485410aa2876" %}
+
+package:
+  name: nifty
+  version: {{ version }}
+
+source:
+    url: https://github.com/DerThorsten/nifty/archive/{{ version }}.tar.gz
+    sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+  skip: True  # [py2k]
```

This is always the case no that conda-forge has stopped building for Python 2.7:
```suggestion
  number: 0
```